### ISSUE TITLE
New addon: exact costume positioning

### ISSUE THREAD

**Issue by lisa-wolfgang** [ScratchAddons/ScratchAddons#3445]
### Is your request related to a problem? Please describe.

When editing costumes in the costume editor, the strange movement intervals when using the arrow keys at various zoom levels and/or imprecision of the mouse can cause an element to not position exactly where desired. This results in some costumes having a messy and slightly unaligned appearance.

### Describe the solution you'd like

When enabled, this addon would add the following input fields to edit the X and Y coordinates of the currently selected element, similar to how the Outline input field functions:

![image](https://user-images.githubusercontent.com/43426138/132275845-f55cfae6-26c8-4c13-aa4a-8dd456394508.png)

### Describe alternatives you've considered

The addon could also simply make the element snap to whole pixel coordinates, which could be adjusted in the settings, but a feature to directly enter specific coordinates seems more useful. It would also make for a better addon thumbnail 😛

**Comment by retronbv** [comment]
What about a rotation input?

**Comment by ErrorGamer2000** [comment]
Good idea, this could be really useful. I have been frustrated by this very part of Scratch *many* times.

**Comment by TheColaber** [comment]
I also suggested a height and width input somewhere... I'm not sure if I just pr'd or created an issue.

**Comment by mxmou** [comment]
@TheColaber #1837

**Comment by Samq64** [comment]
A feedback comment suggested putting the height/width inputs next to the shape being drawn instead of in the toolbar.
